templates/aruba_os_show_aaa_authentication_all.template

```diff
@@ -0,0 +1,12 @@
+Value METHOD (Web|802.1x|Management|tacacs-accounting)
+Value SUCCESSES (\d+)
+Value FAILURES (\d+)
+
+
+Start
+  ^Auth Method Statistics -> AUTHMETHODSTATS
```

Comment:
I don't think the state change does a lot for us, but if you want to keep it, then that is fine. I would ask that it follows TextFSMs state conventions of CamelCase

Reply:
Sure, in the limited context that you guys use the template.  However, I also use these to parse pieces out of “show tech-support” that also include these commands, need the state change to keep from matching very similar tables in output.  Same goes for /S.

Reply:
I don't see how the templates would be reliable if you are feeding the entire show-tech output. I think you would have to split the file into separate command entries or else you  will eventually end up including data from other commands, or miss subtle differences in command output between different OS versions and platforms.